docs/playbooks/intro/README.md

```diff
@@ -8,7 +8,7 @@ If you don't have infrakit or go compiler installed locally, just
 
 ```shell
 
-docker run --rm -v `pwd`:/build infrakit/installer build-infrakit darwin
+docker run --rm -v `pwd`:/build -e GOARCH=amd64 -e GOOS=darwin infrakit/installer build-infrakit
```

Comment:
This change seems to be from a commit that was merged into master the other day... How come it's showing up in your PR?   Can you rebase?

Reply:
Doh, tried using the GitHub Desktop tool to make my workflow a little easier and it does some weird syncing/merging of commits.